Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.cs

```diff
@@ -130,7 +130,7 @@ internal class Nested
     {
         private class PrivateClass;
 
-        protected enum ProtectedEnum { }
+        protected enum ProtectedEnum;
```

Comment:
Interesting, I recall having problems removing brackets for enums in a recent PR, but it does work.

Reply:
Because I disabled RCS1055 ;-). I'll try to make that clear with the rationale for that commit.